polybar-scripts/wakatime-status/README.md

```diff
@@ -0,0 +1,51 @@
+# Polybar WakaTime status
+
+![](/img/screenshot.png)
+
+Display your daily coding time in [Polybar](https://github.com/polybar/polybar).
+## Prerequisites
+- [WakaTime](wakatime.com) account and API token 
+- Polybar 
+
+## Dependencies
+
+- [jq](https://stedolan.github.io/jq/) for JSON parsing in bash
+- [openssl](https://www.openssl.org/)
+- curl 
+
+## Security 
+
+There is provision for you to store your WakaTime API key as plain text in the script file but this is not recommended. 
+
+If you wish to encrypt your password and store it separately in another file there is a [helper script](./scripts/token_encrypt.sh) that does this for you. If you choose this option the script will source your password and decrypt it before making the request to WakaTime. This isn't 100% secure but it is safer than storing you unencrypted token in the Polybar script. Obviously store your public password and the file containing the encrypted token in another non-Git directory.  
```

Comment:
> This isn't 100% secure but it is safer than storing you unencrypted token in the Polybar script.

This is just wrong. You add no security here. You add complexitiy by introducing a mechanism to encrypt the token. But you put the clear text key also in a file. And probably the file is in the same directory like the script.

Its like putting the key of your door under the doormat. Keeping the door unlocked would be the same level of security.

The best way to solve this is to keep the used token in the script. Every user can decide how to change it then, like replacing the line with a password manager query that returns the token.

Reply:
Thanks for the helpful feedback. Understood. This PR removes the security options and leaves security up to the user as advised. 

Yes it would be great if you could link the source repo in the README, cheers.